### PR TITLE
Bills branch

### PR DESCRIPTION
**Pull request by hubi0111**
add officer card

### CODE REVIEW

File: page-officers.hbs [hunk 1]
```diff
@@ -313,6 +313,15 @@ into the {body} of the default.hbs template --}}
                     <a target="_blank" class="icon-github" href="https://github.com/johnlee-jh"></a>
                   </div>
                 </div>
+                <div class="ieee-officer-card">
+                  <img src="{{asset 'officers/Website/bill_hu.png'}}" alt="profilepic"/>
```
Comment (PotatoParser) [inline page-officers.hbs]: The file you added was a `.jpg`, not a `.png`

**Review by PotatoParser — CHANGES_REQUESTED** [review]
Looks very good!

**Review by PotatoParser — APPROVED** [review]
Looks good now!

**Review by encadyma — APPROVED** [review]
Looks great!

**Pull request merged**